README.md

```diff
@@ -102,6 +107,9 @@ If you're running the command as part of an automated build on an unattended bui
 
 You may need to restart the agent service or the computer before the environment variables are available to the agent.
 
+### Docker containers
+You may use the example Dockerfile provided here: https://gist.github.com/shubham90/ad85f2546a72caa20d57bce03ec3890f
```

Comment:
@shubham90 are you happy having this link here or should we create and example file in the repo?

Reply:
Probably better to have an example file in the repo - do you think the helpers folder is sufficient or maybe a samples folder?

Reply:
I'll make it a samples folder. Maybe there are other samples that would be helpful.

Reply:
done.